guardiansFrontend/src/index.js

```diff
@@ -5,15 +5,28 @@ import App from './App';
 import { Route, Link, BrowserRouter as Router } from 'react-router-dom'
 import 'bootstrap/dist/css/bootstrap.css';
 import './Components/css/teamsComp.css'
-import TeamsComponent from './Components/TeamsComponent'
-const routing = (
-  
+import TeamsComponent from './Components/TeamsComponent';
+import createTeam from './Components/createTeam';
+import oneteam from './Components/showOneTeam';
+import upadteTeam from './Components/updateTeam';
+import ParticipantsComponent from './Components/ParticipantsComponent';
+import createParticipant  from "./Components/createParticipant";
+import showOneParticipant from './Components/showOneParticipant';
+import updateParticipant from './Components/updateParticipant';
 
+const routing = (
     <Router>
     <div>
       <Route path="/" component={App} />
       <Route path="/teams" component={TeamsComponent} />
       <Route path="/themes"/>
+      <Route path="/team/create/" component={createTeam}/>
+      <Route path="/team/" component={oneteam}/>
+      <Route path='/update/team/' component={upadteTeam} />
+      <Route path="/participants" component={ParticipantsComponent}/>
+      <Route path="/participant_/create/" component={createParticipant}></Route>
+      <Route path="/participant/" component={showOneParticipant}></Route>
+      <Route path="/participant__/update/" component={updateParticipant}></Route>
```

Comment:
@aluno19426  Cria rotas mais amigáveis para os devs plz Something like this: http://example.comt/{base-path}/{resource-path} . A pasta dos node_modules deve estar no gitignore senão é só tralha @JMendes1995